src/storage_persister.rs

```diff
@@ -1,16 +1,30 @@
 use bitcoin::hashes::hex::ToHex;
+use lightning::chain;
+use lightning::chain::chaininterface::{BroadcasterInterface, FeeEstimator};
 use lightning::chain::chainmonitor::MonitorUpdateId;
 use lightning::chain::chainmonitor::Persist;
 use lightning::chain::channelmonitor::ChannelMonitor;
 use lightning::chain::channelmonitor::ChannelMonitorUpdate;
-use lightning::chain::keysinterface::Sign;
+use lightning::chain::keysinterface::{KeysInterface, Sign};
 use lightning::chain::transaction::OutPoint;
 use lightning::chain::ChannelMonitorUpdateErr;
+use lightning::ln::channelmanager::ChannelManager;
+use lightning::routing::gossip::NetworkGraph;
+use lightning::routing::scoring::WriteableScore;
+use lightning::util::logger::Logger;
+use lightning::util::persist::Persister;
 use lightning::util::ser::Writeable;
+use std::io::Error;
+use std::ops::Deref;
 
 use crate::callbacks::RedundantStorageCallback;
 
 static MONITORS_BUCKET: &str = "monitors";
+static OBJECT_BUCKET: &str = "objects";
```

Comment:
If we follow monitors bucket naming:
```suggestion
static OBJECTS_BUCKET: &str = "objects";
```